src/main/java/stsjorbsmod/patches/WrathField.java

```diff
@@ -0,0 +1,14 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpireField;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+
+@SpirePatch(
+        clz = AbstractCard.class,
+        method = SpirePatch.CLASS
+)
+public class WrathField {
+    @SuppressWarnings("unchecked")
+    public static SpireField<Integer> upgradeCount = new SpireField(() -> 0);
```

Comment:
nit: the word `upgrade` is a little overloaded as applied to cards, and the field name here is what actually gets injected as the field name on the card, so it will show up in debugging. Maybe `wrathDamageIncrease` would be more clear?

Reply:
I want to indicate the number of times Wrath affected it instead of the damage increase. Maybe `wrathDamageInstances` or `wrathApplicationCount`. It will be something.